src/api/app/views/webui2/webui/project/_tabs.html.haml

```diff
@@ -18,11 +18,10 @@
       - unless project.defines_remote_instance?
         %li.nav-item
           = tab_link('Users', project_users_path(project))
-      // This link is intentionally hidden, only users that access to the staging workflow show will see it
-      // TODO: show it if the project have a staging_workflow attached when this feature will be released
-      - if controller_path.starts_with?('webui/staging/')
+      - active = controller_path.starts_with?('webui/staging')
+      - if !project.staging_project? || active
         %li.nav-item
-          = tab_link('Staging', request.url, true)
+          = tab_link('Staging', new_staging_workflow_path(project: project), active)
```

Comment:
Can you move this to the end? Only a very minimal set of projects will have a staging. Other things are more important.

Reply:
@hennevogel  You are right! Staging tab is now the last element in the tablist and the gif is updated. :ok_hand: